packages/frontend/scripts/sync-fathom-events/index.ts

```diff
@@ -44,7 +44,7 @@ const syncFathomEvents = async (siteId?: string, authToken?: string) => {
 
         const syncedEvents = [...fathomSiteEvents, ...newSiteEvents];
         await writeFile(
-            `${__dirname}/../../src/analytics/generated.ts`,
```

Comment:
WHat about having a `generated` folder with a `fathom` module in it instead? The idea being that if in the future we need to have more autogenerated files we can just put them there. It also makes it easier to ignore generated code as we can ignore the entire folder instead of each single file (and as such we should update gitignore, prettierignore and the ESLint config)